gradle/project.gradle.kts

```diff
@@ -142,25 +145,25 @@ tasks {
         exclude("*sources.jar", "*javadoc.jar")
     }
 
-    val jre by creating {
-        val jdkLocal = workDir.resolve("jre")
+    val jdk by creating {
+        val jdkLocal = workDir.resolve("java").toPath()
         group = "build setup"
-        description = "Makes JDK locally accessible in $jdkLocal"
+        description = "Links $jdkLocal to JDK"
         doFirst {
-            if (!jdkLocal.exists()) {
+            if (Files.notExists(jdkLocal)) {
                 println("Making JDK locally accessible...")
-                val jdkGlobalPath = System.getProperty("java.home").takeIf { it.isNotBlank() }
+                val jdk = System.getProperty("java.home").takeIf { it.isNotBlank() }
```

Comment:
Use jdkPath, this is variable shadowing, which is evil. It's bad name anyway.

Reply:
will do